BoxContentSDK/BoxContentSDK/Requests/BOXCollaborationCreateRequest.m

```diff
@@ -8,12 +8,19 @@
 #import "BOXAPIJSONOperation.h"
 #import "BOXCollaboration.h"
 
+@interface BOXCollaborationCreateRequest()
+
+@property (nonatomic, readwrite, strong) BOXAPIItemType *itemType;
```

Comment:
minor - for consistency, either we should have the 2 public readonly properties here, or none (we're missing the item ID). Since we're only manipulating the ivar, it might be okay without ?